### ISSUE TITLE
iptables state fails to save rules 

### ISSUE THREAD

**Issue by rominf** [saltstack/salt#27478]
My state is:
```
iptables_ftp_pasv:
  iptables.append:
    - table: filter
    - chain: INPUT
    - proto: tcp
    - dport: 6980:6985
    - jump: ACCEPT
    - save: True
```

Result is:
```
          ID: iptables_ftp_pasv
    Function: iptables.append
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "/usr/lib/python2.7/site-packages/salt/state.py", line 1561, in call
                  **cdata['kwargs'])
                File "/usr/lib/python2.7/site-packages/salt/states/iptables.py", line 420, in append
                  out = __salt__['iptables.save'](filename, family=family)
                File "/usr/lib/python2.7/site-packages/salt/modules/iptables.py", line 540, in save
                  parent_dir = os.path.dirname(filename)
                File "/usr/lib64/python2.7/posixpath.py", line 129, in dirname
                  i = p.rfind('/') + 1
              AttributeError: 'NoneType' object has no attribute 'rfind'
     Started: 00:21:04.194480
    Duration: 20.835 ms
     Changes:
```

System info:
```
 salt --versions-report
           Salt: 2015.5.5
         Python: 2.7.6 (default, Nov 21 2013, 15:55:38) [GCC]
         Jinja2: 2.7.3
       M2Crypto: 0.22
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.7.0
           RAET: Not Installed
            ZMQ: 4.1.2
           Mako: Not Installed
        Tornado: 4.1
        timelib: Not Installed
       dateutil: 2.4.2
```

```
openSUSE 13.1
Linux 3.11.10-29-default
```

**Comment by jfindlay** [comment]
@rominf, thanks for the report.

**Comment by cachedout** [comment]
Closed via #27515

**Comment by rominf** [comment]
Thank you, it works!
